cpp/ql/src/semmle/code/cpp/models/implementations/Strcat.qll

```diff
@@ -21,7 +21,9 @@ class StrcatFunction extends TaintFunction, DataFlowFunction, ArrayFunction, Sid
         "_mbscat", // _mbscat(dst, src)
         "wcsncat", // wcsncat(dst, src, max_amount)
         "_mbsncat", // _mbsncat(dst, src, max_amount)
-        "_mbsncat_l" // _mbsncat_l(dst, src, max_amount, locale)
+        "_mbsncat_l", // _mbsncat_l(dst, src, max_amount, locale)
```

Comment:
Can you split the charpred into one half with `hasGlobalName` and another with `hasGlobalName`?

Reply:
Oh yeah. I see that I only addressed the use of `getName` in `strcpy`. I can refactor the other ones as well.